ExampleProject/ExampleProject/ExampleProject/CMakeLists.txt

```diff
@@ -22,15 +22,15 @@ if(CMAKE_SIZEOF_VOID_P EQUAL 8)
 	#Link the capstone lib according to config mode
 	target_link_libraries(${PROJECT_NAME} debug ${CAPSTONE_LIB_PATH_64}/Debug/capstone.lib)
 	target_link_libraries(${PROJECT_NAME} optimized ${CAPSTONE_LIB_PATH_64}/Release/capstone.lib)
-	target_link_libraries(${PROJECT_NAME} debug ${PLH_LIB_PATH_64}/Debug/Debug/PolyHook_2.lib)
```

Comment:
Why was this removed?

Reply:
It is not related to the EATHook problem (sorry).

The linking of ExampleProject failed because there is no ${PLH_LIB_PATH_...}/Debug/Debug/PolyHook_2.lib, but there is ${PLH_LIB_PATH_...}/Debug/PolyHook_2.lib.
So I removed one "/Debug" from this line.

(I think the Release has the same issue... Release/Release/PolyHook_2.lib)